website/docs/cheatsheets/python/v1/distributed-training.md

```diff
@@ -66,7 +66,7 @@ Simply ensure that you have taken care of the following:
 * Create an `MpiConfiguration` with your desired distribution.
 
 #### Example
-* [azureml-examples: TensorFlow distributed training using Horovod](https://github.com/Azure/azureml-examples/tree/main/python-sdk/workflows/train/tensorflow/mnist-distributed-horovod)
```

Comment:
Hi @konabuta, links on this page are broken. and I've fixed them in #79 . Could you pull the latest version from main branch?